numpy/testing/__init__.py

```diff
@@ -9,14 +9,14 @@
 
 from unittest import TestCase
 
-from ._private.utils import *
-from ._private import decorators as dec
 from ._private.nosetester import (
     run_module_suite, NoseTester as Tester
     )
+from ._private.utils import *
```

Comment:
Not that I mind, but was there a specific reason for the reordering?

Reply:
I think this was accidental